wgpu/src/backend/webgpu.rs

```diff
@@ -1400,7 +1400,7 @@ impl crate::context::Context for ContextWebGpu {
             feature = "spirv",
             feature = "glsl",
             feature = "wgsl",
-            feature = "naga"
+            feature = "naga-ir"
```

Comment:
this should check the new alias `naga` you just introduced: `not(any(naga, feature = "wgsl"))`

Reply:
Eh, this is basically not the same thing - I'm not predicated on naga, just that none of the following features have been enabled.